smartseq2_single_sample/check_reference/tasks/featurecounts.wdl

```diff
@@ -0,0 +1,46 @@
+task FeatureCountsUniqueMapping {
+  File aligned_bam
+  File gtf
+  String fc_out
+  
+  command {
+    featureCounts -T 4  -s 0 -t exon -g gene_id -p -B -C -a ${gtf} -o "${fc_out}.gene.unq.counts.txt" ${aligned_bam}
+    featureCounts -T 4  -s 0 -t exon -g transcript_id -p -B -C -a ${gtf} -o "${fc_out}.transcript.unq.counts.txt" ${aligned_bam}
+    featureCounts -T 4  -s 0 -t exon -g exon_id -p -B -C -a ${gtf} -o "${fc_out}.exon.unq.counts.txt" ${aligned_bam}
+  }
+  runtime {
+    docker:"humancellatlas/star_dev:v1"
+    memory: "4 GB"
```

Comment:
This cpu count / runtime isn't efficient. 4 cpu should net you 16gb ram. Does this need more than 1 cpu? 3.75 gb is pretty close to 4. :-)

Can also apply to the below task.

Reply:
it could be faster but probably not cheaper